scss/forms/_star-rating.scss

```
@@ -3,31 +3,30 @@
 //
 
 .star-rating {
-  &.disabled,
-  &:disabled {
-    pointer-events: none;
-  }
 
-  > label {
+  > label,
+  span {
     display: block;
     float: left;
     width: 1em;
     height: 1em;
     font-size: $form-star-size;
     cursor: pointer;
-    @include form-star-rating($accessible-orange);
+    @include form-star-rating($accessible-orange, $accessible-orange);
 
     &:not(:first-of-type){
       margin-left: $form-star-margin-between;
     }
   }
 
-  > input:checked ~ label {
-    @include form-star-rating($gray-500);
+  > input:checked ~ label,
+  span {
```

```suggestion
  .checked ~ span {
```
If we want to keep consistency between form and readonly version I think the markup should be:
```html
<div class="star-rating">
  <p class="visually-hidden">Star rating: rated 3 out of 5</p>
  <div aria-hidden="true">
    <span></span>
    <span></span>
    <span class="checked"></span>
    <span></span>
    <span></span>
  </div>
</div>
```
If this is commited and the one below, it might help using both markup anyway.